docs/content/error/$injector/unpr.ngdoc

```diff
@@ -81,3 +81,55 @@ angular.module('myModule', [])
     // a scope object cannot be injected into a service.
   }]);
 ```
+
+Attempting to use unmatched versions of Angular and it's modules will also throw an `Unknown provider` error,
```

Comment:
This is too unspecific; it sounds like mismatched versions will always throw `unpr`. Mixing module versions will only throw `unpr` if a module does not have a provider that is required by another module.